Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -1263,6 +1263,10 @@ void CSystem::CreateSystemVars()
         0,
         "Minimum amount of time to show the level load screen. Important to prevent short loads from flashing the load screen. 0 means "
         "there is no limit.");
+#if defined(CARBONATED)
+    REGISTER_STRING("default_load_screen_config_file", "", 0, "Default Load Screen Config File.");
+    REGISTER_STRING("list_of_levels_without_load_screen", "", 0, "Comma separated list of levels without the level load screen.");
```

Comment:
Per my earlier comment, remove this, and determine whether or not a level has a default based on the presence of their levels/level/level.cfg file.

Reply:
Removed